packages/@uppy/utils/src/Translator.ts

```diff
@@ -96,9 +103,9 @@ function interpolate(
  * Usage example: `translator.translate('files_chosen', {smart_count: 3})`
  */
 export default class Translator {
-  protected locale: Locale
+  locale: Locale
```

Comment:
Why this change? Do we want `locale` to be part of the public API? Shouldn't we at least document it as `readonly`?

Reply:
We access this property outside of Translator so this change was necessary.